projects/canopy/src/lib/table/table/table.component.ts

```diff
@@ -32,12 +36,23 @@ export class LgTableComponent implements AfterContentChecked {
     return this.isExpandable;
   }
 
+  private _columnsBreakpoint: TableColumnLayoutBreakpoints;
```

Comment:
could the private variable shadow the name of the Input?
```suggestion
  private _showColumnsAt: TableColumnLayoutBreakpoints;
```

Reply:
Fixed